modules/katamari/src/main/ts/ephox/katamari/api/Arr.ts

```diff
@@ -247,6 +247,8 @@ export const head = <T>(xs: ArrayLike<T>): Optional<T> => xs.length === 0 ? Opti
 
 export const last = <T>(xs: ArrayLike<T>): Optional<T> => xs.length === 0 ? Optional.none() : Optional.some(xs[xs.length - 1]);
 
+export const get = <T>(xs: ArrayLike<T>, i: number): Optional<T> => i >= 0 && i < xs.length ? Optional.some(xs[i]) : Optional.none();
+
```

Comment:
```suggestion
export const get = <T>(xs: ArrayLike<T>, i: number): Optional<T> => i >= 0 && i < xs.length ? Optional.some(xs[i]) : Optional.none();

export const last = <T>(xs: ArrayLike<T>): Optional<T> => get(xs, xs.length - 1);
```

Reply:
We can reuse `get` now to implement `last`, so I've swapped the order above and made `last` call `get`.

Reply:
Do we want to use it for `head` too?

Reply:
Ahh, I missed that because it's not in the diff. Yes, we should do the same there as well.